package.json

```diff
@@ -6,6 +6,7 @@
     "start": "nx serve",
     "build": "nx build",
     "test": "nx test",
+    "test-changed": "nx affected --base=staging --target=test",
```

Comment:
A contributor usually makes a PR against `staging` (on a fork, we usually don't have `main`, and the command without `--base` fails).

And since `nx` is not installed globally, we need to use it through `pnpm`. So, just doing `nx ...` doesn't work.